Update Readme - add purpose (#90)
appliedblockchain/mantle: README.md

# Mantle

> Mantle is a privacy solution for blockchain platforms like Ethereum. Usually data in private blockchains like Ethereum Poa/Instambul is stored in plain. With Mantle you can store sensible data encrypted and choose a selected number of parites to share this data with. The privacy solution that Mantle gives you sits outside the blockchain meaning that is compatible with blockchains other than Ethereum (like for example Hyperledger Fabric or Corda). 

Reviewer: ```suggestion
Mantle is a privacy solution for blockchain platforms like Ethereum. Usually data in private blockchains like Ethereum Poa/Istanbul is stored in plain. With Mantle you can store sensible data encrypted and choose a selected number of parties to share this data with. The privacy solution that Mantle gives you sits outside the blockchain, meaning that it is compatible with blockchains other than Ethereum (like for example Hyperledger Fabric or Corda). 
```

> Mantle uses an hybrid cryptosystem to achieve so, please look at the SDK repository and documentation for more info.

Reviewer: ```suggestion
Mantle uses a hybrid cryptosystem to achieve this. Please look at the SDK repository and documentation for more info.
```
